lldb/bindings/interface/SBValueDocstrings.i

```diff
@@ -45,6 +45,38 @@ SBValue instance which interprets the value object as representing the head of a
 linked list."
 ) lldb::SBValue;
 
+%feature("docstring", "
+Returns true if the SBValue holds any useful state
+and false otherwise.
+IsValid is a very limited API, lldb will only return
+invalid SBValues if it has no useful information
+about the SBValue.
+The two main ways you will end up with an invalid
+SBValue are:
+1) default constructed SBValues are not valid.
+2) SBValues that have outlived their SBTarget are
+no longer valid since it would not be safe to ask them
+questions. lldb will instead return a default constructed
+return value.  You can tell why this is happening by
+checking IsValid.
```

Comment:
Should that be "by checking GetError"? IsValid doesn't explain why, just that it is invalid, right?